site/kube-addon.md

```diff
@@ -292,3 +292,65 @@ For example,
             - name: IPALLOC_RANGE
               value: 10.0.0.0/16
 ```
+
+## <a name="operations"></a> Operations
+
+Weave Net can be used without Kubernetes, and if you have done this before, you may be familiar with the `weave` CLI.
+
+Within Kubernetes, even though the network peers are managed via Kubernetes and very few commands from the `weave` CLI should be useful, you may still want to run commands to, for example, confirm Weave Net' status. This can be done in various ways:
+
+1. You could [install the `weave` script](/site/installing-weave.md) and run the same commands you would run without Kubernetes:
+
+```
+$ weave status
+        Version: 2.0.1 (up to date; next check at 2017/07/10 13:49:29)
+
+        Service: router
+       Protocol: weave 1..2
+           Name: 42:8e:e8:c4:52:1b(host-0)
+     Encryption: disabled
+  PeerDiscovery: enabled
+        Targets: 3
+    Connections: 3 (2 established, 1 failed)
+          Peers: 3 (with 6 established connections)
+ TrustedSubnets: none
+
+        Service: ipam
+         Status: ready
+          Range: 10.32.0.0/12
+  DefaultSubnet: 10.32.0.0/12
+```
+
+2. If you do not want to install anything else on your hosts, you can use the following `kubectl` commands, which will produce the same outcome as the previous example:
+
+```
+### Identify the Weave Net pods:
+$ kubectl get pods -n kube-system | grep weave-net
+weave-net-1jkl6                  2/2       Running   0          1d
+weave-net-bskbv                  2/2       Running   0          1d
+weave-net-m4x1b                  2/2       Running   0          1d
+
+### Run the commands you want from within one of the pods:
+$ kubectl exec -n kube-system weave-net-1jkl6 -- /home/weave/weave --local status
+Defaulting container name to weave.
+Use 'kubectl describe pod/weave-net-1jkl6' to see all of the containers in this pod.
+
+        Version: 2.0.1 (up to date; next check at 2017/07/10 13:49:29)
+
+        Service: router
+       Protocol: weave 1..2
+           Name: 42:8e:e8:c4:52:1b(host-0)
+     Encryption: disabled
+  PeerDiscovery: enabled
+        Targets: 3
+    Connections: 3 (2 established, 1 failed)
+          Peers: 3 (with 6 established connections)
+ TrustedSubnets: none
+
+        Service: ipam
+         Status: ready
+          Range: 10.32.0.0/12
+  DefaultSubnet: 10.32.0.0/12
+```
+
+3. Finally you could also use [Weave Cloud](https://cloud.weave.works/) and monitor all your pods, including Weave Net's ones, from there.
```

Comment:
Would be interesting to see whether Explore shows the Weave Net topology in this configuration. I think it will.